schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/process/add-extent-from-geokeywords.xsl

```diff
@@ -36,7 +36,7 @@
   <xsl:variable name="replaceMode"
                 select="geonet:parseBoolean($replace)"/>
   <xsl:variable name="serviceUrl"
-                select="concat($gurl, '/keywords?pNewSearch=true&amp;pTypeSearch=2&amp;pKeyword=')"/>
+                select="concat(substring($gurl, 1, string-length($gurl)-4), '/api/registries/vocabularies/search?_content_type=xml&amp;q=')"/>
```

Comment:
You can probably use $catalogUrl ? and apply the same fix in https://github.com/geonetwork/core-geonetwork/blob/master/schemas/iso19139/src/main/plugin/iso19139/process/add-extent-from-geokeywords.xsl#L64

Reply:
There doesn't seem to be a $catalogUrl variable in that XSLT file, it's throwing errors when I make this change without altering the XML.